[BUG] rtl8821cu-kmod fails to build due to missing rtl8821c.mk file

### Full Package Name

rtl8821cu-kmod-5.12.0.4^20251215git.7f63a9d-3.fc43.x86_64

### Release Version

f43

### Describe the Bug

When attempting to build the rtl8821cu-kmod package with akmods, the build fails with a missing file error:

```
Makefile:2389: rtl8821c.mk: No such file or directory
make[3]: *** No rule to make target 'rtl8821c.mk'.  Stop.
```

This appears to be caused by the spec file line:

    cp -fr core hal include os_dep platform Kconfig Makefile halmac.mk _kmod_build_${kernel_version%%___*}

which does not copy rtl8821c.mk (required by the Makefile). The solution seems to be to change it to:

    cp -fr core hal include os_dep platform Kconfig Makefile halmac.mk rtl8821c.mk _kmod_build_${kernel_version%%___*}

### To Reproduce

1. Attempt to build rtl8821cu-kmod via akmods
2. Observe build log for missing file error
3. Confirm that rtl8821c.mk is not copied to the build directory

### Expected Behavior

The build should succeed and not fail due to missing rtl8821c.mk file.

### Relevant Log Output

```shell
Full error (truncated):


Makefile:2389: rtl8821c.mk: No such file or directory
make[3]: *** No rule to make target 'rtl8821c.mk'.  Stop.
make[2]: *** [/usr/src/kernels/6.17.12-300.fc43.x86_64/Makefile:2047: .] Error 2
make[1]: *** [/usr/src/kernels/6.17.12-300.fc43.x86_64/Makefile:248: __sub-make] Error 2
make: *** [Makefile:248: __sub-make] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.r7fgdN (%build)
RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.r7fgdN (%build)


Other (not blocking):
- Compiler mismatch warning
- Occasional sqlite/rpmdb permission errors
```

--- a/anda/system/rtl8821cu/akmod/rtl8821cu-kmod.spec
+++ b/anda/system/rtl8821cu/akmod/rtl8821cu-kmod.spec
@@ -44,7 +44,7 @@ sed -i 's/CONFIG_PLATFORM_ARM64_RPI = n/CONFIG_PLATFORM_ARM64_RPI = y/g' Makefil
 
 for kernel_version in %{?kernel_versions}; do
     mkdir _kmod_build_${kernel_version%%___*}
-    cp -fr core hal include os_dep platform Kconfig Makefile halmac.mk _kmod_build_${kernel_version%%___*}
+    cp -fr core hal include os_dep platform Kconfig Makefile halmac.mk rtl8821c.mk _kmod_build_${kernel_version%%___*}
 done
 
 %build

